src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -501,6 +576,11 @@ ChartsPanel.defaultProps = {
     expandable: true
 };
 
+function getTblIdForChartId(chartId) {
+    return  get(flux.getState()[XYPlotCntlr.XYPLOT_DATA_KEY], [chartId, 'tblId']) ||
+            get(flux.getState()[HistogramCntlr.HISTOGRAM_DATA_KEY], [chartId, 'tblId']);
+}
```

Comment:
this function should be in ChartUtil.  if there is not a ChartUtil, it may be a good idea to create one.  i am sure there will be more of these functions in the future when more chart related features are added.